src/main/java/io/stargate/sgv2/jsonapi/exception/playing/ServerException.java

```diff
@@ -0,0 +1,46 @@
+package io.stargate.sgv2.jsonapi.exception.playing;
+
+/**
+ * Base for any errors that are from the {@link ErrorFamily#SERVER} family, these are server side
+ * errors not related to the structure of the request itself.
+ *
+ * <p>Scope are defined in {@link Scope} and each represents a subclass of this class.
+ *
+ * <p>See {@link APIException}
+ */
+public class ServerException extends APIException {
+
+  public static final ErrorFamily FAMILY = ErrorFamily.SERVER;
```

Comment:
This is problematic:

1. Should not really be `public` -- why would it be needed from outside (probably `protected` for sub-classes)
2. Name is very vague, esp. when referred to from sub-classes. Maybe `SERVER_EXCEPTION_FAMILY`
3. ... or is this constant needed at all? Just use `ErrorFamily.SERVER` directly in sub-class.

Reply:
idea was for subclasses to all reference the same definition of the family during construction, e.g. in the ctor for the Code enum